### DISCUSSION TITLE
web analytics or tag manager

### DISCUSSION THREAD

**Discussion by marco-santulli** [nocodb/nocodb#891] [Features]
Hi,

is there a way to add a tag manager or other web analytics to noco? I really like the fact we can add sentry but would be good to be able to easily add other tools?

thx

Marco

**Comment by o1lab** [comment]
Not available, its on the frontend and to add it dynamically we would have to think about it.

**Reply by marco-santulli** [reply]
I think even something like adding a NC_HEADER_SNIPPET file parameter (a bit like the NC_DB_JSON_FILE parameter) and then inject it as high in the head as possible would be good enough to get people going.

**Reply by marco-santulli** [reply]
do you think it is doable?
